devito/function.py

```diff
@@ -950,6 +950,11 @@ def __init__(self, *args, **kwargs):
             else:
                 raise TypeError("`space_order` must be int or 3-tuple of ints")
 
+            # Symbolic (finite difference) coefficients
+            self._coefficients = kwargs.get('coefficients', 'standard')
```

Comment:
Isn't this a bit "cyclic". Need to declare function with symbolic coefficient and need function to declare coefficients?

Reply:
The declaration of symbolic in the function allows for the 'potential' use of symbolic coefficients. It maybe that the user wants to use custom coefficients for a certain derivative in a particular dimension or just in a certain equation (using standard coefficients in another equation). Therefore, without making further changes to the manner in which the fd coefficients are implemented, we need something of this form to allow this flexibility. 

Note that if the coefficients function is not declared (but the functions coefficients have been declared symbolic) the symbols will be replaced with the default weights.